src/styles/transitions.js

```diff
@@ -15,6 +15,19 @@ export const easing = {
   sharp: 'cubic-bezier(0.4, 0.0, 0.6, 1)',
 };
 
+// Follow https://material.io/guidelines/motion/duration-easing.html#duration-easing-common-durations
```

Comment:
Created some constants for transition durations, but on second thought, i think theese should be subject of customization/theme somehow - even documentation gives different durations for different devices (pc, tablet, mobile, watches)